### ISSUE TITLE
panel-tabs with lots of tabs is not responsive

### ISSUE THREAD

**Issue by leolivier** [jgthms/bulma#3866]
### Overview of the problem
This is about the Bulma **CSS framework** 
I'm using Bulma **version** [1.0.1]
My **browser** is: Firefox and Edge
I am sure this issue is **not a duplicate**

### Description
I have the following html piece:
```html
<section class="panel">
		<p class="panel-heading">Statistics</p>
		<div class="panel-tabs">
			<a class="stat-category is-active" data-category="site">Site</a>
			<a class="stat-category " data-category="members">Members</a>
			<a class="stat-category " data-category="galleries">Galleries</a>
			<a class="stat-category " data-category="forums">Forums</a>
			<a class="stat-category " data-category="chats">Chats</a>
			<a class="stat-category " data-category="administrator">Administrator</a>
		</div>
		<div class="panel-category" id="category-site" style="">
			<div class="panel-block">
			...
```
When displayed on a mobile phone, what I see is
![image](https://github.com/user-attachments/assets/38c2d034-c4d9-4fc3-b3db-8352fd08a4c2)

So I don't see the "Site" tab on the left, and I cannot swipe left to try to see it.
(I can swipe right and see "Administrator" in full)

### Steps to Reproduce

See above

### Expected behavior

At least, be able to swipe left.
I tried to change the justify-content from center to left and then I can see Sites and swipe right up to "Administrator"

Or maybe restack the tabs vertically when they don't fit horizontally?

**Comment by leolivier** [comment]
FYI, I just added in my "custom.css" the lines and it solved the issue
```css.
panel-tabs {
	flex-wrap: wrap;
}
```
I don't know if it should applied to whole Bulma...

**Comment by jgthms** [comment]
Yeah it's a tricky one because some people prefer scrolling (which would require a parent element to be scrollable), some people prefer wrapping like you did.

I'll close this for now because I think it's better to leave it to each developer but thanks for the update.

**Comment by guest20** [comment]
Maybe a class="tabs does-linewrap" could be worth including in the library?

I have some css I copy/pasted from an earlier issue #1344 to make my tavs wrap...

It might be worth including a default one way or the other, since the current default is a 3rd, unusable choice where the items aren't visible and can't be accessed (until someone reports a bug)

**Comment by jgthms** [comment]
You can already use the Flexbox helper `is-flex-wrap-wrap`: https://bulma.io/documentation/helpers/flexbox-helpers/

**Comment by guest20** [comment]
TL;DR: Thanks for the CSS! Maybe overflow:hidden isn't the best default for tabs? Maybe `.scrolling-tabs` and `,wrapping-tabs` or a docpatch would help the css-illiterate?

@jgthms I am a perl developer. User-journey-wise, I'm definitely at the "I don't know what a flex box is" end of the spectrum. I already struggle to plan how my layout will look at different break points, though bulma definitely shoves my stuff a notch closer to usable on my (aging, janky) mobile device, so, thanks loads for the free css. It's very helpful and good. Would recommend to a friend or coworker.

Without discounting that, it feels when selecting between "scrollable", "wrapping", and "making some of your navigation inaccessible", the latter is the least desirable of the 3 choices, and maybe shouldn't be the default? 

I'm betting that there's a deep-css-magic-based reason to not pick one, but in a case like that it would be very helpful to have a kind of "⚠ this component requires you make a choice" marker in the docs. Maybe even with an "if you don't know/care, just use me with `is-flex-wrap-wrap`". 

I'd even be willing to make my users download the extra bytes for a `class=wrapping-tabs` and a `class=scrolling-tabs` in addition to the regular old classic `class=tabs`
